### DISCUSSION TITLE
Cronical folder purpose

### DISCUSSION THREAD

**Discussion by phamducphutrong** [leokhoa/laragon#646] [Q&A]
I installed laragon I want to delete some folders that I don't wanna use. However, there is a folder called cronical, I don't know what is the purpose of this folder. Please help me?

**Comment by Pen-y-Fan** [comment]
It's for running cron jobs on Windows, the directory can be deleted if you don't use it.

See https://pen-y-fan.github.io/2023/01/28/how-to-run-cron-jobs-on-windows/ for more details.
